src/app/client/src/app/modules/dashboard/components/addusers/addusers.component.ts

```diff
@@ -5058,21 +5058,12 @@ alert(e)
     }
```

Comment:
@snehalsabade Is this MR is reverted na?

Reply:
Yes mam reverted